roles/appsuite/templates/opt/open-xchange/etc/findbasic.properties.j2

```diff
@@ -0,0 +1,18 @@
+{{ ansible_managed | comment }}
+
+# Some mail backends provide a virtual folder that contains all messages of
+# a user to enable cross-folder mail search. Open-Xchange can make use of
+# this feature to improve the search experience.
+#
+# Set the value to the name of the virtual mail folder containing all messages.
+# Leave blank if no such folder exists.
+com.openexchange.find.basic.mail.allMessagesFolder = Virtual/All
```

Comment:
Please make this one a template and set a default of `''` (empty string) in the role defaults.